.github/libs/GithubUtils.ts

```diff
@@ -58,12 +105,13 @@ class GithubUtils {
      * @static
      * @memberof GithubUtils
      */
-    static get octokit() {
+    static get octokit(): RestEndpointMethods {
         if (this.internalOctokit) {
             return this.internalOctokit.rest;
         }
         this.initOctokit();
-        return this.internalOctokit.rest;
+        // @ts-expect-error -- by running this.initOctokit() above we can be sure that this.internalOctokit is defined
+        return this.internalOctokit.rest as RestEndpointMethods;
     }
```

Comment:
If we invert the condition we can improve this a little bit:

```ts
    static get octokit(): RestEndpointMethods {
        if (!this.internalOctokit) {
            this.initOctokit();
        }

        // eslint-disable-next-line @typescript-eslint/non-nullable-type-assertion-style
        return (this.internalOctokit as InternalOctokit).rest;
    }
```

Reply:
Much better, thanks!